Reorganize documentation (#23)
liteflow-labs/docs: services/http.md

# HTTP

> The HTTP Service let you create an HTTP endpoints and and connect every requests of the HTTP to endpoint to any process of your choice.

Reviewer: ```suggestion
The HTTP Service lets you create an HTTP endpoint that triggers the associated process.
```

## Events

### ![trigger](/trigger.svg) Request

#### Definition

> By setting the parameter `endpointKey`, the process will be triggered every time a POST request is made to a dedicated and generated URL.

Reviewer: ```suggestion
The parameter `endpointKey` helps to generate a unique URL. Keep the same value when you update your process to keep the same URL. Change its value to generate a new unique URL.
```
